op.cpp

```diff
@@ -36,6 +36,7 @@ double operation (char op, double a, double b) 						//Check which operation and
 		return a+b;
 	if (op == '-')
 		return a-b;
+	return 0;
```

Comment:
0 (and all values) are valid return values; defaulting to 0 on an invalid operand can mask unexpected results